types.d.ts

```diff
@@ -160,9 +160,14 @@ declare namespace BLECentralPlugin {
         ): Promise<void>;
         stopNotification(device_id: string, service_uuid: string, characteristic_uuid: string): Promise<void>;
 
-        /* Returns a rejected promise if the device is not connected */
+        /* Returns a resolved promise if the device is connected,
+            otherwise returns rejected promise if the device is not connected */
         isConnected(device_id: string): Promise<void>;
 
+        /* Returns a promise that resolves to true if the device is connected,
+            otherwise resolves to false if the device is not connected or an error occurs */
+        isConnected(device_id: string, rejectWhenDisconnected: false): Promise<boolean>;
```

Comment:
Type of argument rejectWhenDisconnected should be boolean instead of false

Reply:
By deliberately setting it to `false`, I can narrow the return type  to `Promise<boolean>`.

Calling this with `true` is effectively the same as the `resolve` or `reject` overload above.

Reply:
Yep, sorry
You are right 
I missed the definition without the second argument 
isConnected(device_id: string): Promise<void>;